crates/blockchain-tree/src/lib.rs

```diff
@@ -56,4 +56,6 @@ pub mod noop;
 
 mod state;
 
+pub use state::BlockchainId;
```

Comment:
I'd like to keep this type private or pub crate

Reply:
Thanks @mattsse I will fix,  I'm curious about the reasoning behind this.

Reply:
this type is only used internally and should not be exposed